src/app/components/Trackcard(tracklist)/Trackcard-tracklist.module.css

```diff
@@ -4,28 +4,26 @@
   background-color: var(--background-color);
   box-shadow: -0.3em -0.3em 0.5em var(--text-color);
   max-width: 95%;
-  max-height: 12rem;
+  max-height: 3.5rem;
   display: grid;
-  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
   grid-template-rows: 1fr 1fr;
   grid-template-areas:
-    'cardimage cardimage title title title likebutton likebutton'
-    'cardimage cardimage artist artist artist likebutton likebutton';
+    'cardimage cardimage title title title title likebutton likebutton'
+    'cardimage cardimage artist artist artist artist likebutton likebutton';
```

Comment:
haven't seen this before, so not sure but this looks like something that can be shortend?

Reply:
It can't be shortened because those are grid template areas  which assign a designated space to my items in a grid, like you can see in my code. 
![grid](https://user-images.githubusercontent.com/77332349/123419882-11d98200-d5bb-11eb-894e-6135391fd285.png)